components/renku_data_services/message_queue/avro_models/io/renku/events/v1/project_created.py

```diff
@@ -20,6 +20,7 @@ class ProjectCreated(AvroModel):
     repositories: List[str]
     visibility: Visibility
     description: Optional[str]
+    keywords: List[str]
```

Comment:
```suggestion
    keywords: List[str] = field(default_factory=list)
```

See the other comment.